src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -95,6 +95,19 @@ Robocopy "%PACKAGES_DIR%\MicroBuild.Core\%MICROBUILD_VERSION%\build\." "%TOOLRUN
 Robocopy "%PACKAGES_DIR%\Microsoft.Net.Compilers\%ROSLYNCOMPILERS_VERSION%\." "%TOOLRUNTIME_DIR%\net46\roslyn\." /E
 
 @echo on
+call "%DOTNET_CMD%" build "%TOOLRUNTIME_DIR%\ilasm\ilasm.depproj" -r win-x64 --source https://dotnet.myget.org/F/dotnet-core/api/v3/index.json --packages "%PACKAGES_DIR%\."
```

Comment:
While I agree x64 is the common default there might be issues with anyone that may or may not be using this in Razzle because that requires x86 tools.

Reply:
> using this in Razzle

Is there anybody still using recent enough buildtools in Razzle?

Reply:
I hope not but I'm not entirely sure. My main point is that there are some envirments that still force x86 toolsets. I think it is OK to assume x64 but we may need to figure out a way to make it an option somehow. I think my other suggestion about moving this into the il.targets instead might help mitigate that issue as then we can set an msbuild property to control it if needed.